docs/how-to/ccv0.sh

```diff
@@ -44,16 +44,26 @@ if [ -z ${GOPATH} ]; then
     export GOPATH=${HOME}/go
 fi
 export tests_repo_dir="${GOPATH}/src/${tests_repo}"
+echo ${tests_repo_dir}
 export katacontainers_repo_dir="${GOPATH}/src/${katacontainers_repo}"
 export ROOTFS_DIR="${katacontainers_repo_dir}/tools/osbuilder/rootfs-builder/rootfs"
 export PULL_IMAGE="${PULL_IMAGE:-quay.io/kata-containers/confidential-containers:signed}" # Doesn't need authentication
 export CONTAINER_ID="${CONTAINER_ID:-0123456789}"
 source /etc/os-release || source /usr/lib/os-release
 grep -Eq "\<fedora\>" /etc/os-release 2> /dev/null && export USE_PODMAN=true
 
-export BATS_TEST_DIRNAME="${tests_repo_dir}/integration/containerd/confidential"
+
 # If we've already checked out the test repo then source the confidential scripts
-[ -d "${BATS_TEST_DIRNAME}" ] && source "${BATS_TEST_DIRNAME}/lib.sh"
+if [ "${KUBERNETES}" == "no" ]; then
```

Comment:
If `KUBERNETES` isn't set then it will fall into `else` which actually set variables for the Kubernetes case and you don't want that, right? I would invert the logic to `if [ "${KUBERNETES:-}" == "yes" ] ... kubernetes ... else .... non-kubernetes`